www/page_data.inc

```diff
@@ -457,6 +457,15 @@ function loadPageStepData($localPaths, $testInfo = null) {
       }
     }
 
+    // set a top level firstContentfulPaint metric

```

Comment:
This needs to be done before line 304 so the value is available for the seek start and also around 504 in case the page data is being loaded from a cached copy (for previous tests).

Reply:
I _think_ we're in better shape here. Tested with cache on and off and getting FCP for both.